judge-d-server/src/test/resources/swagger-deploy-sphere-api.json

```diff
@@ -0,0 +1,2166 @@
+{
+  "swagger" : "2.0",
```

Comment:
Please fix swagger so they it covers only the scenario you would like to test in your test case